app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeFriendUi.kt

```diff
@@ -33,12 +33,14 @@ import com.github.se.eventradar.R
 import com.github.se.eventradar.ui.BottomNavigationMenu
 import com.github.se.eventradar.ui.navigation.NavigationActions
 import com.github.se.eventradar.ui.navigation.TOP_LEVEL_DESTINATIONS
+import com.github.se.eventradar.viewmodel.MyQrCodeViewModel
 import com.github.se.eventradar.viewmodel.qrCode.ScanFriendQrViewModel
 
 // TODO cleaner code for Navigation and to correct screen
 
 @Composable
 fun QrCodeScreen(
+    myQrCodeViewModel: MyQrCodeViewModel = hiltViewModel(),
     viewModel: ScanFriendQrViewModel = hiltViewModel(),
```

Comment:
It is very unconventional to have two view models for one screen. If there is any way to combine these, I'm all for it.